__tests__/app.test.js

```diff
@@ -34,4 +34,26 @@ describe('GET Requests', () => {
             });
         });
     });
+    test('/api/articles 200: responds with all articles & associated comment counts', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((res) => {
+            //Check Correct Length
+            expect(res.body.articles).toHaveLength(13);
+            //Check Sort Order
+            expect(res.body.articles[0].title).toBe("Eight pug gifs that remind me of mitch");
+            expect(res.body.articles[12].title).toBe("Z");

```

Comment:
This can and should be more dynamic, I would recommend looking into the jest-sorted library on a nicer and robust way to do this.